release-notes/README.md

```diff
@@ -4,6 +4,12 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## June 17, 2021
```

Comment:
I believe this was released on June 16th, @ravjain-adb can you confirm?

Reply:
17 is good we did android on 16 and ios on 17.

Reply:
or we can split this in two parts one for iOS and one for Android ? I am ok this way too.

Reply:
Yup we can divide the release. 
@spoorthipujariadobe lets make android at 16 and ios at 17